components/buttons/DocsButton.js

```diff
@@ -0,0 +1,64 @@
+import Link from 'next/link';
+
+export default function DocsButton({ suggestions, parentClass }) {
```

Comment:
We should avoid `parentClass` name, please use `className` :)

Reply:
> We should avoid `parentClass` name, please use `className` :)

Yepp, I will change this.